Terminal.Gui/Views/TableView/TreeTableSource.cs

```diff
@@ -0,0 +1,203 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+
+namespace Terminal.Gui;
+
+/// <summary>
+/// An <see cref="ITableSource"/> with expandable rows.
+/// </summary>
+/// <typeparam name="T"></typeparam>
+public class TreeTableSource<T> : IEnumerableTableSource<T>, IDisposable where T : class {
+	private TreeView<T> tree;
```

Comment:
Please prefix privates with `_` and don't use `this.`.

Reply:
Fixed in [bc14970](https://github.com/gui-cs/Terminal.Gui/pull/2685/commits/bc14970f1c7580ffd6a48661d635623187aaa1af)